front-end/src/components/Post/GovernanceSideBar/VoteReferendum.tsx

```diff
@@ -102,39 +101,6 @@ const VoteRefrendum = ({ className, referendumId, api, apiReady, address, defaul
 
 	return (
 		<div className={className}>
-			{/* <div className='card'>
```

Comment:
put that code in the issue where it might come back.
https://github.com/paritytech/polkassembly/issues/306#issuecomment-593983103